src/pages/billing/Project.js

```diff
@@ -27,9 +41,51 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [billingAccountName, setBillingAccountName] = useState(null)
   const [showBillingModal, setShowBillingModal] = useState(false)
   const [selectedBilling, setSelectedBilling] = useState()
+  const [workspaces, setWorkspaces] = useState(() => StateHistory.get().projectUsers || null)
+  const [tab, setTab] = useState(query.tab || 'workspaces')
 
   const signal = Utils.useCancellation()
 
+  const tabToTable = {
+    workspaces: div({ style: { flexGrow: 1 } }, [
+      _.map(workspace => h(WorkspaceCard, { workspace }), workspaces)
+    ]),
+    users: h(Fragment, [
+      h(NewUserCard, {
+        onClick: () => setAddingUser(true)
+      }),
+      div({ style: { flexGrow: 1 } },
+        _.map(member => {
+          return h(MemberCard, {
```

Comment:
I believe you will need to provide a `key` here and at line 51 for the mapped over elements to prevent the warning:
`Warning: Each child in a list should have a unique "key" prop.`